process/working_groups.md

```diff
@@ -6,24 +6,27 @@ parent: Processes
 # ASWF Working Groups
 _Modeled off of CNCF Working Group model ( https://github.com/cncf/toc/blob/master/workinggroups/README.md )_
 
-## Introduction
-
 The purpose of working groups are to study and report on a particular question and make recommendations based on it's findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing it's findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a [TAC member] and with an affirmative vote of the ASWF TAC. The TAC can also shut down a working group with a super majority vote.
 
-## Process
+## Forming a working group
+
+Interested parties in forming a working groups should align with the interested parties on the [charter template]. Once the working group charter is agreed upon, and a [TAC member] is identified as a sponsor, it should be shared with the [TAC email list] to be added to the agenda for the next TAC meeting for review and approval. A working group will not be designated and recognized as an official ASWF Working Group until the TAC approves the charter.
```

Comment:
I'm not sure I understand that first sentence. Are there two sets of "interested parties"? Also "forming a working groups" should either be "forming a working group" or "forming working groups", but probably the former. How about:

Parties interested in forming a working group should use the [charter template] to define a charter to be proposed to the TAC.

Does that capture it?

Reply:
Fixed now - thanks!